packages/plugin/webpack/src/Config.ts

```diff
@@ -84,6 +84,12 @@ export interface WebpackPluginConfig {
    * The webpack config for your main process
    */
   mainConfig: WebpackConfiguration | string;
+  /**
+   * In the event that webpack has been configured with `devtool: sourcemap` (or any other option
+   * which results in a `.map` file being generated), this option will cause it to not be
+   * packaged with your app.
+   */
+  ignoreSourcemap?: boolean;
```

Comment:
```suggestion
  ignoreSourceMap?: boolean;
```

nit: The spec seems to define it as source map rather than sourcemap https://sourcemaps.info/spec.html

Would be good to have correct capitalization out the gate.